crates/pixi_consts/src/consts.rs

```diff
@@ -51,6 +51,10 @@ pub const PIXI_DIR: &str = match option_env!("PIXI_DIR") {
     Some(dir) => dir,
     None => ".pixi",
 };
+pub const MANIFEST_DEFAULT_NAME: &str = match option_env!("PIXI_MANIFEST_DEFAULT_NAME") {
```

Comment:
Could you add some documentation for this feature?

Reply:
Also the env var name should include "global" somewhere.

Reply:
Done!